nx/lib/nx/defn/graph_splitter.ex

```diff
@@ -0,0 +1,283 @@
+defmodule Nx.Defn.GraphSplitter do
```

Comment:
```suggestion
defmodule Nx.Defn.Graph do
```

And traverse becomes `split`?